tests/piqueserver/test_extensions.py

```diff
@@ -27,24 +37,20 @@ def test_checkscripts(self):
         self.assertFalse(extensions.check_scripts(scripts))
 
     def test_load_scripts(self):
-        # Define scripts
-        script_names = ['test_script_game_mode', 'test_script_regular']
-        script_names_length = len(script_names)
-        # Get current directory
-        curr_dir = os.getcwd()
-        script_dir = os.path.join(curr_dir, 'piqueserver/test_scripts/')
+        self.assertTrue(len(self.script_objects) == self.script_names_length)
+        # apply_script method should exist in all modules
+        for i in range(0,len(self.script_objects)):
```

Comment:
remember to follow pep8, ideally use a checker like `flake8` or a fixer like `autopep8`